### PR TITLE
changed computation of iWat in TWetBul_TDryBulXi for #759

### PR DESCRIPTION
**Pull request by Mathadon**
for #759

### CODE REVIEW

File: IBPSA/Utilities/Psychrometrics/TWetBul_TDryBulXi.mo [hunk 1]
```diff
@@ -4,8 +4,7 @@ block TWetBul_TDryBulXi
   extends Modelica.Blocks.Icons.Block;
 
   replaceable package Medium =
-    Modelica.Media.Interfaces.PartialCondensingGases "Medium model"
-                                                            annotation (
+    IBPSA.Media.Air "Medium model"                          annotation (
```
Comment (mwetter) [inline IBPSA/Utilities/Psychrometrics/TWetBul_TDryBulXi.mo]: Why is this change needed? Neither Dymola nor JModelica needs it.

**Comment by Mathadon** [comment]
If I don't change it, then I get:
screen shot 2017-05-01 at 15 26 56

**Comment by mwetter** [comment]
@Mathadon : Is this a problem, as you check the block with unassigned medium? The following works in Dymola 2017 FD01 with the pedantic mode:
```
checkModel("IBPSA.Utilities.Psychrometrics.Examples.TWetBul_TDryBulXi");
```
We have the same issue with any model in `.Fluid` unless a medium is assigned.

**Comment by Mathadon** [comment]
I get the error in the screenshot and 
```
Advanced.PedanticModelica:=true;
checkModel("IBPSA.Utilities.Psychrometrics.TWetBul_TDryBulXi");
 = false
```
with `Medium = Modelica.Media.Interfaces.PartialCondensingGases` and no errors with `IBPSA.Media.Air`. 

So I understand you don't get these errors in 2017FD01? So I should revert back to the old Medium?

**Comment by Mathadon** [comment]
Yes, when I check the block with the default Medium then I get that error.

The example is fine. The old implementation didn't have that error so I wanted to avoid it, but I did indeed not realise that there are more models that have this effect when the medium is unassigned.

I'll revert the medium change.
